src/api/RebalancingAPI.ts

```diff
@@ -184,10 +184,41 @@ export class RebalancingAPI {
    * @param  bidQuantity                    Amount of currentSet the bidder wants to rebalance
    * @return                                Transaction hash
    */
-  public async getBidPriceAsync(rebalancingSetTokenAddress: Address, bidQuantity: BigNumber): Promise<TokenFlows> {
+  public async getBidPriceAsync(
+    rebalancingSetTokenAddress: Address,
+    bidQuantity: BigNumber,
+  ): Promise<TokenFlowsDetails> {
     await this.assertGetBidPrice(rebalancingSetTokenAddress, bidQuantity);
 
-    return await this.rebalancingSetToken.getBidPrice(rebalancingSetTokenAddress, bidQuantity);
+    const tokenFlowComponents = await this.rebalancingSetToken.getBidPrice(rebalancingSetTokenAddress, bidQuantity);
+    const tokenAddresses = await this.rebalancingSetToken.getCombinedTokenArray(rebalancingSetTokenAddress);
+
+    const inflow = tokenFlowComponents.inflow.reduce((acc, unit, index) => {
+      const bigNumberUnit = new BigNumber(unit);
```

Comment:
the return value from the contract wrapper is already a big number I believe

Reply:
apparently the contract wrapper converts it into string?

TypeError: unit.gt is not a function

      196 |     const inflow = tokenFlowComponents.inflow.reduce((accumulator, unit, index) => {
      197 |       const bigNumberUnit = new BigNumber(unit);
    > 198 |       if (unit.gt(0)) {
      199 |         accumulator.push({
      200 |           address: tokenAddresses[index],
      201 |           unit,